### ISSUE TITLE
models: bibrec table still used?

### ISSUE THREAD

**Issue by jmartinm** [inveniosoftware/invenio-records#5]
I am wondering what the plan is for the `bibrec` table?

When I run [$ inveniomanage create](https://github.com/inveniosoftware/invenio-records/blob/master/invenio_records/api.py#L70) I am getting both `record_json` and `bibrec` populated but all the entries in `bibrec` look like:

```
mysql> select * from bibrec limit 5;
+----+---------------------+---------------------+---------------+-----------------+
| id | creation_date       | modification_date   | master_format | additional_info |
+----+---------------------+---------------------+---------------+-----------------+
|  1 | 1900-01-01 00:00:00 | 1900-01-01 00:00:00 | marc          | NULL            |
|  2 | 1900-01-01 00:00:00 | 1900-01-01 00:00:00 | marc          | NULL            |
|  3 | 1900-01-01 00:00:00 | 1900-01-01 00:00:00 | marc          | NULL            |
|  4 | 1900-01-01 00:00:00 | 1900-01-01 00:00:00 | marc          | NULL            |
|  5 | 1900-01-01 00:00:00 | 1900-01-01 00:00:00 | marc          | NULL            |
+----+---------------------+---------------------+---------------+-----------------+
5
```

1) Are we going to continue using that table?

2) Should all the metadata about the record be populated properly in `bibrec`?

PS. Actually I don't even find where the `bibrec` table is being updated, on the `create` function I only see the `RecordMetadata` table being updated [here](https://github.com/inveniosoftware/invenio-records/blob/master/invenio_records/api.py#L87)

**Comment by jirikuncar** [comment]
It's related to PIDStore. You don't need to worry about it so far ...

**Comment by tiborsimko** [comment]
(FWIW another thing to mention related to the `bibrec` table is inherent requirement of using integer record ID values.  Thanks to the use of Elasticsearch, this will become unnecessary in the future.  bibrec table could later go, and we could enable non-numeric record IDs.  This would require to clean all our codebase of recID-based assumptions... Let's see after module separation.)

**Comment by lnielsen** [comment]
Solved in 1.0.0 (i.e. bibrec is removed)
